vendor/github.com/ethereum/go-ethereum/rpc/client.go

```diff
@@ -488,6 +488,11 @@ func (c *Client) write(ctx context.Context, msg interface{}) error {
 	c.writeConn.SetWriteDeadline(deadline)
 	err := json.NewEncoder(c.writeConn).Encode(msg)
 	if err != nil {
+		// If write timed out do not try to establish a new connection next time
```

Comment:
I'm guessing this commit was inadvertently included

Reply:
Ah was using this in local tests - will remove for now